web/src/app/opening/openings.component.html

```diff
@@ -29,9 +52,12 @@ <h2>{{ 'OPENING.LIST' | translate }}</h2>
                         <td>{{opening.modifiedDate | date: 'dd-MM-yyyy'}}</td>
                         <td>
                             <button *ngIf="loggedRole == 'Sr.HR'" title="Delete" class="btn btn-danger no-radius" (click)="deleteOpening(opening.openingId)">{{ 'OPENING.DELETE' | translate }}</button>
-                            <button *ngIf="loggedRole == 'Sr.HR' && opening.status == 'Re-Edit'" title="Edit" class="btn btn-primary no-radius blue-btn" (click)="updateOpening(opening.openingId)">{{ 'OPENING.UPDATE' | translate }}</button>
-                            <button *ngIf="loggedRole != 'Sr.HR' && !opening.isApproved" title="Edit Opening" class="btn btn-primary no-radius blue-btn" (click)="updateOpening(opening.openingId)">{{ 'OPENING.EDIT' | translate }}</button>
-                            <button *ngIf="loggedRole == 'Sr.HR' && opening.isApproved" title="AddCandidate" class="btn btn-primary no-radius blue-btn" (click)="addCandidate(opening.openingId)">{{ 'OPENING.ADD' | translate }}</button>
+                            <button *ngIf="loggedRole == 'Sr.HR' && opening.status == 'Re-Edit'" title="Edit" class="btn btn-primary no-radius blue-btn"
+                                (click)="updateOpening(opening.openingId)">{{ 'OPENING.UPDATE' | translate }}</button>
+                            <button *ngIf="loggedRole != 'Sr.HR' && !opening.isApproved" title="Edit Opening" class="btn btn-primary no-radius blue-btn"
+                                (click)="updateOpening(opening.openingId)">{{ 'OPENING.EDIT' | translate }}</button>
+                            <button *ngIf="loggedRole == 'Sr.HR' && opening.isApproved" title="AddCandidate" class="btn btn-primary no-radius blue-btn"
+                                (click)="addCandidate(opening.openingId)">{{ 'OPENING.ADD' | translate }}</button>
```

Comment:
*ngIf="loggedRole != 'Sr.HR' wrong way, find out generic way for such conditions.

Reply:
Done